### DISCUSSION TITLE
Help me. I can't solve the unkown

### DISCUSSION THREAD

**Discussion by mineclover** [openapi-ts/openapi-typescript#1391] [General]
https://api.redocly.com/registry/bundle/xsolla/igs-bb-api/v1/openapi.yaml

I received a d.ts file through In this api specification basic usage

version >>> "openapi-typescript": "^7.0.0-next.2",
script >>> openapi-typescript openapi/spec.yaml --output src/types/api.ts

From this file
type Test = paths['/v2/project/{project_id}/items']['get']['responses']['200']['content']['application/json']['items']

on the receiving end of
if (props == null) return null
if ('item_id' in props) return <div>{props.item_id}</div>
Props.item_id is treated as unkown when done

<img width="1077" alt="image" src="https://github.com/drwpow/openapi-typescript/assets/61359316/0bae7d82-a5dd-4e9f-96f4-ae614fda6ec4">

I solved it thank!

**Comment by drwpow** [comment]
Ah glad you figured it out. What was the issue?

**Reply by mineclover** [reply]
The open api document I used was a payment document... 
I was using a modular component, a web trend, so I had to put NonNullable where I had to control null
